### ISSUE TITLE
build failed

### ISSUE THREAD

**Issue by akinschen** [vrtdev/flutter_workmanager#62]
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/apple/git_flutter/flutter/.pub-cache/hosted/pub.flutter-io.cn/workmanager-0.1.0/ios/Classes/SwiftWorkmanagerPlugin.swift:98:156: error: argument type 'FlutterEngine' does not conform to expected type 'FlutterBinaryMessenger'
            var backgroundMethodChannel: FlutterMethodChannel? = FlutterMethodChannel(name: BackgroundMethodChannel.channelName, binaryMessenger: flutterEngine!)
                                                                                                                                                  ~~~~~~~~~~~~~^
                                                                                                                                                                 as! FlutterBinaryMessenger

**Comment by akinschen** [comment]
Which flutter version is **flutter_workmanager** support?

**Comment by Kymer** [comment]
We use `Flutter 1.7.8+hotfix.4 • channel stable`. Can you run `flutter doctor` and report back?

**Comment by Kymer** [comment]
@akinschen This has been fixed in version 0.1.0 of the plugin 👍
